nncf/config_schema.py

```diff
@@ -231,6 +232,40 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
     "additionalProperties": False,
 }
 
+RANGE_INIT_CONFIG = {
```

Comment:
Looks like this is a duplicate. Should remove the original, if you plan on extracting these piece to reuse.

Reply:
`RANGE_INIT_CONFIG_PROPERTIES`

Reply:
Agreed, done